echopype/consolidate/api.py

```diff
@@ -227,77 +234,52 @@ def add_location(
     echodata : EchoData or str or pathlib.Path
         An ``EchoData`` object or path to a file containing the ``EchoData``
         object holding the raw data
-    nmea_sentence
-        NMEA sentence to select a subset of location data (optional)
+    datagram_type : Optional[str], default None
+        The type of datagram used to select latitude and longitude.
+        Can only be used for EK sonar models and value can be either 'NMEA', 'MRU1' or 'IDX'.
+    nmea_sentence : Optional[str], default None
+        NMEA sentence to select a subset of location data
```

Comment:
```suggestion
    datagram_type : Optional[str], default None
        Datagram type to use for latitude and longitude.
        Can be 'NMEA', 'MRU1' or 'IDX'.
        Can only be used for EK sonar models.
    nmea_sentence : Optional[str], default None
        NMEA sentence to select a subset of location data
```

Reply:
Ah I see. I didn't check for `datagram_type == 'NMEA'` in the code since that would be the `else` (default) case. I agree though that making it explicit is the way to go.